These global symbols are used within libmandb, so if we want to make it
shared, they need to be moved from each of the executables into the library

--- origsrc/man-db-2.6.7/libdb/Makefile.am	2013-12-08 18:43:01.000000000 -0600
+++ src/man-db-2.6.7/libdb/Makefile.am	2014-05-30 12:02:03.758696100 -0500
@@ -43,4 +43,4 @@ libmandb_la_SOURCES = \
 
 libmandb_la_LIBADD = ../lib/libman.la $(DBLIBS)
 
-libmandb_la_LDFLAGS = -avoid-version -release $(VERSION) -rpath $(pkglibdir)
+libmandb_la_LDFLAGS = -avoid-version -release $(VERSION) -rpath $(pkglibdir) -no-undefined
--- origsrc/man-db-2.6.7/libdb/db_ver.c	2013-12-08 18:38:13.000000000 -0600
+++ src/man-db-2.6.7/libdb/db_ver.c	2014-05-30 12:03:48.648695400 -0500
@@ -37,6 +37,10 @@
 
 #include "mydbm.h"
 
+/* globals */
+char *database = NULL;
+MYDBM_FILE dbf;
+
 int dbver_rd (MYDBM_FILE dbfile)
 {
 	datum key, content;
--- origsrc/man-db-2.6.7/src/accessdb.c	2014-01-13 05:18:41.000000000 -0600
+++ src/man-db-2.6.7/src/accessdb.c	2014-05-30 12:12:09.086318800 -0500
@@ -49,10 +49,6 @@
 char *program_name;
 const char *cat_root;
 
-/* for db_storage.c */
-char *database;
-MYDBM_FILE dbf;
-
 const char *argp_program_version = "accessdb " PACKAGE_VERSION;
 const char *argp_program_bug_address = PACKAGE_BUGREPORT;
 error_t argp_err_exit_status = FAIL;
--- origsrc/man-db-2.6.7/src/catman.c	2014-01-14 20:22:15.000000000 -0600
+++ src/man-db-2.6.7/src/catman.c	2014-05-30 12:12:18.804874700 -0500
@@ -82,10 +82,8 @@
 /* globals */
 char *program_name;
 int quiet = 1;
-MYDBM_FILE dbf;
 char *manp;
 extern char *user_config_file;
-char *database;
 static const char **sections;
 
 const char *argp_program_version = "catman " PACKAGE_VERSION;
--- origsrc/man-db-2.6.7/src/man.c	2014-03-26 06:19:14.000000000 -0500
+++ src/man-db-2.6.7/src/man.c	2014-05-30 18:01:03.559596500 -0500
@@ -188,8 +188,6 @@ static char *manpathlist[MAXDIRS];
 /* globals */
 int quiet = 1;
 char *program_name;
-char *database = NULL;
-MYDBM_FILE dbf; 
 extern const char *extension; /* for globbing.c */
 extern char *user_config_file;	/* defined in manp.c */
 extern int disable_cache;
--- origsrc/man-db-2.6.7/src/mandb.c	2014-04-09 19:30:39.000000000 -0500
+++ src/man-db-2.6.7/src/mandb.c	2014-05-30 12:11:38.595574900 -0500
@@ -71,9 +71,7 @@
 char *program_name;
 int quiet = 1;
 extern int opt_test;		/* don't update db */
-MYDBM_FILE dbf;
 char *manp;
-char *database = NULL;
 extern char *extension;		/* for globbing.c */
 extern int force_rescan;	/* for check_mandirs.c */
 static char *single_filename = NULL;
--- origsrc/man-db-2.6.7/src/whatis.c	2014-01-13 05:18:41.000000000 -0600
+++ src/man-db-2.6.7/src/whatis.c	2014-05-30 12:12:26.972341900 -0500
@@ -85,8 +85,6 @@ static int num_keywords;
 
 char *program_name;
 int am_apropos;
-char *database;
-MYDBM_FILE dbf;
 int quiet = 1;
 
 #ifdef HAVE_ICONV
